nginx/nginx.conf

```diff
@@ -0,0 +1,31 @@
+events {
+}
+
+http {
+    server_tokens off;
+    charset utf-8;
+        upstream frontend {
+            server 127.0.0.1:3000;
+          }
+    server {
+        listen 80;
+        listen 443 ssl http2 default_server;
+        listen [::]:443 ssl http2 default_server;
+        ssl_certificate /etc/ssl/certs/nginx-selfsigned.crt;
+        ssl_certificate_key /etc/ssl/private/nginx-selfsigned.key;
```

Comment:
Running it on port 80 is good enough, right? Do we need 443 for the localhost?

Reply:
we need it to connect redirection from login page(for instance) to the dev server app.